cypress/e2e/tests/upgrade/after_upgrade.test.ts

```diff
@@ -135,6 +140,34 @@ describe(["@post-upgrade"], "Performing post-upgrade validations", () => {
         sourceApplication.selectApplication();
     });
 
+    it("Verify that assessed application exists and is assessed", function () {
+        const assessmentApplication = new Analysis(
```

Comment:
We should be able get Application object by just name . No need for these steps .